lib/partitioninfo.coffee

```diff
@@ -92,6 +102,21 @@ callWithDisk = (fn, pathOrDisk, args...) ->
 
 ###*
 # @summary Get information from a partition
+#
+# @description `getPartitions()` returns an Array.
```

Comment:
Looks like some parts of the `@description` here should be in the jsdoc for `exports.getPartitions`, rather than `exports.get` as it is now?